cdap-runtime-ext-dataproc/src/main/resources/gcp-dataproc.json

```diff
@@ -257,6 +266,20 @@
         }
       ]
     },
+    {
+      "label": "GCP CMEK Information",
```

Comment:
since there's only one property now, I don't think we need a separate group, can just put it above the GCS bucket